data/tickers.js

```diff
@@ -1,58 +1,15 @@
+const faker = require('faker');
+
 module.exports = {
   createTickers() {
-    // returns the same array of 100 unique strings (tickers)
-      // each ticker is 3 - 4 characters in length // TODO: vary the ticker length
-      // each ticker is made up of the letters A - Z
-
-      let seed = 1;
-      let result = [];
-      let ticker = [];
-
-      // start at decimal 65
-      let startValue = 65;
-      let decimal = 65;
-      let incrementor = 0;
-      let letter = 0;
-
-      for (let i = 0; i < 100; i++) {
-        for (let j = 0; j < 4; j++) {
-          if (decimal + j < 65 || decimal + j > 90) {
-            incrementor++;
-            seed = 1;
-            decimal = startValue;
-          }
-
-          if (j === 2 && incrementor === 1) {
-            ticker.reverse();
-          }
-
-          letter = String.fromCodePoint(decimal + j);
-
+    let result = [];
 
-          // every time the incrementor changes, build the ticker in a different order
-          if (incrementor === 0) {
-            ticker.push(letter);
-          } else if (incrementor === 1) {
-            ticker.unshift(letter);
-          } else if (incrementor === 2) {
-            ticker.splice(Math.floor(ticker.length / 2), 0, letter);
-          } else {
-            if (incrementor === 3) {
-              ticker.splice(0, 0, letter);
-            } else if (j === 1) {
-              ticker.splice(1, 0, letter);
-            } else if (j === 2) {
-              ticker.splice(1, 0, letter);
-            } else if (j === 3) {
-              ticker.splice(2, 0, letter);
-            }
-          }
-        }
-        decimal += seed;
+    for (let i = 0; i < 1000; i++) {
```

Comment:
Oh I see, you're batching the tickers here... How can you be sure that you're generating enough unique tickers?

Reply:
Good point! I know I set the ticker to be unique on my schema so there wouldn't be any duplicate entries in the DB

Reply:
I double-checked and I do see 1000000 rows in my table